cli/src/pixl_cli/_config.py

```diff
@@ -13,20 +13,66 @@
 #  limitations under the License.
 
 """Configuration of CLI from config file."""
-from pathlib import Path
+from typing import Optional
 
-import yaml
+from decouple import config
 
+SERVICE_SETTINGS = {
```

Comment:
One thing here, I don't think any of these should have defaults that aren't set in the `.env.sample`, because then if I forget to set one it'll look like I have the config set up

Reply:
Makes sense, the only annoying thing is that you will then have to source the `.env` file first every time before running any of the `pixl` CLI commands 😕

Reply:
Why would you have to do that? Doesn't the PIXL CLI read in the .env file as part of the config?

Reply:
Don't think it does? At least not when I run it from the `test/` directory in the system-test:

```sh
+ pixl populate /Users/milan/Projects/UCLH-Foundry/PIXL/test/resources/omop
Traceback (most recent call last):
  File "/Users/milan/.pyenv/versions/3.10.13/envs/pixl-3.10.13/bin/pixl", line 5, in <module>
    from pixl_cli.main import cli
  File "/Users/milan/Projects/UCLH-Foundry/PIXL/cli/src/pixl_cli/__init__.py", line 17, in <module>
    from pixl_cli.main import cli
  File "/Users/milan/Projects/UCLH-Foundry/PIXL/cli/src/pixl_cli/main.py", line 28, in <module>
    from pixl_cli._config import SERVICE_SETTINGS, api_config_for_queue
  File "/Users/milan/Projects/UCLH-Foundry/PIXL/cli/src/pixl_cli/_config.py", line 21, in <module>
    "host": config("RABBITMQ_HOST"),
  File "/Users/milan/.pyenv/versions/pixl-3.10.13/lib/python3.10/site-packages/decouple.py", line 243, in __call__
    return self.config(*args, **kwargs)
  File "/Users/milan/.pyenv/versions/pixl-3.10.13/lib/python3.10/site-packages/decouple.py", line 105, in __call__
    return self.get(*args, **kwargs)
  File "/Users/milan/.pyenv/versions/pixl-3.10.13/lib/python3.10/site-packages/decouple.py", line 90, in get
    raise UndefinedValueError('{} not found. Declare it as envvar or define a default value.'.format(option))
decouple.UndefinedValueError: RABBITMQ_HOST not found. Declare it as envvar or define a default value.

```

Reply:
So what I've done now is load the `.env` file from the current working directory with

```python
# Make sure the local .env file is loaded
load_dotenv(Path.cwd() / ".env")
```

so I think that's the second option you suggested?